### DISCUSSION TITLE
Create Block variation in custom block

### DISCUSSION THREAD

**Discussion by mKult-admin** [WordPress/gutenberg#64587] [General]
In Edit i tried to set ALLOWED_BLOCKS to the name of the block variation. In the editor i get no result. If set ALLOWED_BLOCKS to the core/block name i get the original block and the variation. 
Is there a way to get only the variation?

**Comment by stokesman** [comment]
> Is there a way to get only the variation?

Not that I'm aware of. Perhaps this would be worth making an issue that suggests it as an enhancement.

A possible workaround might be having the custom block transform the core block into the variation anytime one is inserted within but that’s probably less than ideal.
